orderer/common/cluster/util.go

```diff
@@ -48,6 +48,17 @@ type ConnByCertMap map[string]*grpc.ClientConn
 // Lable used for TLS Export Keying Material call
 const KeyingMaterialLabel = "orderer v3 authentication label"
 
+//go:generate mockery --dir . --name LedgerWriter --case underscore --output mocks/
```

Comment:
I don't think they belong to util. These are definitions.

Let's put them in the file they're used at. which is probably deliver.go.

Reply:
LedgerWriter is only used by LedgerInterceptor which is in the same file.
Am I missing something?

https://github.com/hyperledger/fabric/blob/c31a8a23ae414a1209c9019bf51120d03a21bded/orderer/common/cluster/util.go#L588

Reply:
You're not, I didn't look at the code when reviewing. 

But now when I look at the code (at least, one version of it...)
Is this ledger interceptor in use after removing onboarding?

Reply:
You are correct, the LedgerWriter is used by LedgerInterceptor which is only by the test, removed both.